### PR TITLE
Set `status` as default metricset in Apache module

### PR DESCRIPTION
**Pull request by ruflin**
The metricset config was removed from the short config as there is only 1 metricset anyway. No need for the user to configure this.

### CODE REVIEW

File: metricbeat/docs/modules/apache.asciidoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ This file is generated! See scripts/docs_collector.py
 == Apache module
 
 This module periodically fetches metrics from https://httpd.apache.org/[Apache
-HTTPD] servers.
+HTTPD] servers. The default metricset is `namespace`.
```
Comment (jsoriano) [inline metricbeat/docs/modules/apache.asciidoc]: Defalt metricset is status :)

Comment (ruflin) [reply metricbeat/docs/modules/apache.asciidoc]: copy/paste :-( Fixed.

File: metricbeat/module/apache/_meta/config.yml [hunk 2]
```diff
@@ -1,5 +1,4 @@
 - module: apache
-  metricsets: ["status"]
   period: 10s
 
   # Apache hosts
```
Comment (andrewkroh) [inline metricbeat/module/apache/_meta/config.yml]: I think we could drop the "Apache hosts" comment. It really doesn't any new info IMO.

Comment (ruflin) [reply metricbeat/module/apache/_meta/config.yml]: host comment removed and also period removed.

**Pull request merged**